translations/base-no.yaml

```diff
@@ -30,59 +30,90 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        #shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
```

Comment:
Comments inside multi-line strings are not allowed in YAML. They are interpreted as part of the string.
Please either remove them, or move them outside the string.
```
#longText: >-
#  original english
longText: >-
  translated text
```